protocol-dashboard/index.html

```diff
@@ -3,10 +3,10 @@
 
 <head>
   <meta charset="utf-8" />
-  <link rel="shortcut icon" href="favicon.ico" />
+  <link rel="shortcut icon" href="/favicon.ico" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#CC0FE0" />
-  <link rel="manifest" href="manifest.json" />
+  <link rel="manifest" href="/manifest.json" />
```

Comment:
i think this will cause it to break on prod. i'm okay with merging and finding out, but i removed the leading `/` before because i think when it builds in prod and then is served at `/ipfs/<cid>` then the `/` makes it try to read /manifest.json when it should really be `/ipfs/<cid>/manifest.json`

Reply:
ok gotcha... was this issue introduced recently or has it always been around?

Reply:
there was no issue before. then, there was an issue when switching to vite with base not being set (defaulting to `/`). then, i fixed it by setting base to `./` and making the manifest and favicon not be absolute (removing the `/` prefix in index.html)

Reply:
https://github.com/AudiusProject/audius-protocol/pull/6270/commits/bc957e2803a7ef6aca5a7c67f69f564ba66d55d4 and https://github.com/AudiusProject/audius-protocol/pull/6270/commits/915d56632d2376fbd80d228b780fb578c0e897e8 i think should do this while accommodating the variable base path